LSP.sublime-settings

```diff
@@ -78,11 +78,6 @@
   // See also: "diagnostics_delay_ms".
   "diagnostics_additional_delay_auto_complete_ms": 0,
 
-  // Highlighting style of code diagnostics.
-  // Valid values are "fill", "box", "underline", "stippled", "squiggly" or "".
-  // When set to the empty string (""), no diagnostics are shown in-line.
-  "diagnostics_highlight_style": "squiggly",
```

Comment:
Would anyone be using `""`? I'm gambling no.

Reply:
Yes, me. It's sometimes better to have gutter icons only. It doesn't distract that much as all the inline highlighting which updates too late or is normal while statements are still incomplete.

Reply:
`show_diagnostics_highlights: true/false` then

Reply:
Can't you use `show_diagnostics_severity_level` for that (set it to 0)?

Reply:
Oh, that also hides the gutter icon...